libs/shared/src/constants.ts

```diff
@@ -39,3 +39,7 @@ export const MAX_TRUNCATED_CONTENT_LENGTH = 2_000;
 
 export const TEST_BLOCK_INFO_STRING =
   '{"number":1,"hash":"0x0f927bde6fb00940895178da0d32948714ea6e76f6374f03ffbbd7e0787e15bf","timestamp":1665083987891}';
+
+export const MIN_CHARS_TO_SHOW_MORE = 500;
+export const MAIDEN_CHARS_TO_SHOW_MORE = 1000;
+export const MAX_CHARS_TO_SHOW_MORE = 5000;
```

Comment:
The maximum length of bodies stored in the database is 2000. See `MAX_TRUNCATED_CONTENT_LENGTH` above. Any threads that contain more than 2000 characters have the full body stored on R2 and this body is not loaded until the user opens the thread page. May need to reduce this value to 2000 and check if `content_url` is set to know if there are more than 2k characters.

I think it is important for @mzparacha to review this PR since he worked on the client side for loading thread bodies that are longer than 2k characters.